frontend/taipy-gui/src/components/Taipy/Input.tsx

```diff
@@ -66,6 +66,7 @@ const Input = (props: TaipyInputProps) => {
         multiline = false,
         actionOnBlur = false,
         linesShown = 5,
+        size = "medium",
```

Comment:
If you want to support size in input, it has to be defined on the python side too:
[here](https://github.com/Avaiga/taipy/blob/6c236cb8551765658402f6b5143be2c96505f725/taipy/gui/_renderers/factory.py#L333)
and as a property definition [there](https://github.com/Avaiga/taipy/blob/6c236cb8551765658402f6b5143be2c96505f725/taipy/gui/viselements.json#L148)

You can copy the size property definition from button